SQL/Archive/2018-04-06_ImprovedDescriptionOfLookupCenterNameUsing.sql

```diff
@@ -0,0 +1,5 @@
+-- Change description and label of the ConfigSetting named lookupCenterNameUsing
+UPDATE ConfigSettings SET
+    Description = "DICOM field to use to get the patient and center name of the DICOM study, being either PatientID or PatientName",
```

Comment:
I would rephrase to say "the patient identifier (being either the PatientName or PatientID), and center name...."

Reply:
aaaaah, I was looking for that "identifier" word but it never came to my brain... Thank you!! Changing it right away :)